### PR TITLE
Add creation time to job status

### LINKED ISSUE #1217: Print allocation creation time with 'nomad status'.
I think it would be useful to print the allocation creation time in the allocations section with the ‘nomad status ’ command which currently prints "ID|Eval ID|Node ID|Task Group|Desired|Status"

### PR DESCRIPTION
**Pull request by a86c6f7964**
closes #1217 

I decided to take the approach of adding a command line flag to add the extra metadata, the other idea would be to make -verbose add the extra metadata.

### CODE REVIEW

**Comment by diptanu** [comment]
@a86c6f7964 Let's show created time by default since this essentially shows how long the allocation has been running for.

**Comment by a86c6f7964** [comment]
@diptanu if an allocation is retried the creation time does not change. That table is already pretty wide if you turn on `-verbose` are we really sure that we want to add created to it for everything even though you can see much better information about start/stop/restart time in the nomad alloc-status command?

**Comment by diptanu** [comment]
@a86c6f7964 Yeah it is ok to add the created time in the nomad status, as it provides a sense of the age of the allocation which is operationally relevant most of the time.

File: command/status.go [hunk 1]
```diff
@@ -364,6 +358,12 @@ func (c *StatusCommand) outputFailedPlacements(failedEval *api.Evaluation) {
 	}
 }
 
+// formatUnixNanoTime is a helper for formatting time for output.
+func (c *StatusCommand) formatUnixNanoTime(nano int64) string {
```
Comment (dadgar) [inline command/status.go]: Move this to same file as `formatTime`

**Comment by dadgar** [comment]
Please update these two pages as well:

https://www.nomadproject.io/intro/getting-started/jobs.html
https://www.nomadproject.io/docs/commands/status.html

**Comment by a86c6f7964** [comment]
okay I made changes from the feedback. thanks for the help. do you think that we should add creation time to the allocations that `nomad node-status` returns as well, since it now always showing up for allocations?

**Comment by diptanu** [comment]
@a86c6f7964 Yes that would be nice, and in addition to the `node-status` we should have this in `alloc-status` too.

**Comment by a86c6f7964** [comment]
are you waiting on something from me? Do you only want to merge this once the other issues are complete?

File: command/status.go [hunk 2]
```diff
@@ -319,15 +301,16 @@ func (c *StatusCommand) outputJobInfo(client *api.Client, job *api.Job) error {
 	c.Ui.Output(c.Colorize().Color("\n[bold]Allocations[reset]"))
 	if len(jobAllocs) > 0 {
 		allocs = make([]string, len(jobAllocs)+1)
-		allocs[0] = "ID|Eval ID|Node ID|Task Group|Desired|Status"
+		allocs[0] = "ID|Eval ID|Node ID|Task Group|Desired|Status|Created"
```
Comment (dadgar) [inline command/status.go]: Can we change `Created` to `Created At`

**Comment by dadgar** [comment]
LGTM. One small cosmetic comment

**Comment by a86c6f7964** [comment]
changed to "Created At"

**Comment by dadgar** [comment]
Nice thank you!

**Comment by diptanu** [comment]
@a86c6f7964 Thanks, merging this one! Are you planning to tackle the two issues anytime soon?

**Comment by dadgar** [comment]
We merged within 10 seconds of each other!

**Comment by a86c6f7964** [comment]
I would like to work on the other issues next, but I have been trying to get tests to pass.

**Pull request merged**